includes/Story_Post_Type.php

```diff
@@ -518,7 +522,19 @@ public function get_editor_settings() {
 			$max_upload_size = 0;
 		}
 
-		$is_demo = ( isset( $_GET['web-stories-demo'] ) && (bool) $_GET['web-stories-demo'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$is_demo          = ( isset( $_GET['web-stories-demo'] ) && (bool) $_GET['web-stories-demo'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$all_stories_link = add_query_arg(
+			[
+				'post_type' => self::POST_TYPE_SLUG,
+			],
+			admin_url( 'edit.php' )
+		);
```

Comment:
This should probably link to the dashboard instead

Reply:
Changed in https://github.com/google/web-stories-wp/pull/6497/commits/68a86af6150b025eec38d036c5bc1733f40dc2dd